src/p_map.c

```diff
@@ -1772,7 +1772,7 @@ static boolean P_SplashImmune(mobj_t *target, mobj_t *spot)
     mobjinfo[target->type].splash_group == mobjinfo[spot->type].splash_group;
 }
 
-static boolean PIT_RadiusAttack(mobj_t *thing)
+boolean PIT_RadiusAttack(mobj_t *thing)
```

Comment:
Good approach - we should also do this for PIT_ChangeSector above to allow PIT_ChangeSector to be referenced in p_maputil.c.